source/module_io/input.cpp

```diff
@@ -297,9 +297,12 @@ void Input::Default(void)
     // potential / charge / wavefunction / energy
     //----------------------------------------------------------
     init_wfc = "atomic";
+    psi_initializer = false;
+    wfc_dump = false;
     mem_saver = 0;
     printe = 100; // must > 0
     init_chg = "atomic";
+    init_wfc = "atomic";
```

Comment:
Line 299 has another init_wfc?

Reply:
sorry I didnt realize that, I have delete this one